src/config/index.ts

```diff
@@ -168,10 +169,27 @@ export const SENTRY_REPOS: Set<string> = new Set([
 export const GH_USER_TOKEN = process.env.GH_USER_TOKEN || '';
 
 /**
- * Load GitHubOrgs. We support processing events coming at us from multiple
- * GitHub orgs and this is how we encapsulate it all.
+ * Load GitHubOrgs. We [want to] support processing events coming at us from
+ * multiple GitHub orgs and this is how we [plan to] encapsulate it all.
  */
 export const GH_ORGS: GitHubOrgs = loadGitHubOrgsFromEnvironment(process.env);
+export const GETSENTRY_ORG =
+  process.env.GH_APP_IDENTIFIER && process.env.GH_APP_SECRET_KEY
```

Comment:
What do you think about this logic for creating the placeholder org to live in `loadGitHubOrgsFromEnvironment`? There seems to be an conditional case there for `process.env.GH_APP_IDENTIFIER && process.env.GH_APP_SECRET_KEY`, so we can use that so this can be shortened to 

`GETSENTRY_ORG = GH_ORGS.get(process.env.GETSENTRY_ORG || 'getsentry')`

Reply:
But then wouldn't it return the stub object for _any_ org slug? The purpose here is to populate a stub object for only `getsentry`, since that is hardwired around the app. We're okay with the app failing moderately opaquely at runtime if `GETSENTRY_ORG` is not configured, because there shouldn't be any GitHub events reaching those parts of the app that aren't relevant outside `getsentry`. I can add this in a comment if helpful.

Reply:
Done in ae3f5bcc8bf8c4ba6c6b424191fca82d02272fe8. Look better?

Reply:
fair enough